scripts/generate_random_grids.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Generate random grids/graphs with obstacles.
+This script can optionally suppress the output
+display of the grid and graph nodes.
+The generated graphs are saved as .npz files.
+"""
+
+import argparse
+from my_research.utils.grid_dijkstra import generer_grille, save_graph
```

Comment:
I don't see any module named "my_research" in your PR. Are you sure it's the right path?